mundane-bytes/src/main/scala/com/ambiata/mundane/bytes/Buffer.scala

```diff
@@ -0,0 +1,26 @@
+package com.ambiata.mundane.bytes
+
+case class Buffer(bytes: Array[Byte], offset: Int, var length: Int) {
```

Comment:
> Would consider adding skull and cross bones.

Is there any sense in calling it "MutableBuffer" or "UnsafeBuffer", or is that dodging the issue and a big comment is all we need?